server/monitor/src/main/java/org/apache/accumulo/monitor/Monitor.java

```diff
@@ -642,7 +643,7 @@ private void getMonitorLock() throws KeeperException, InterruptedException {
       UtilWaitThread.sleep(getContext().getConfiguration().getTimeInMillis(Property.MONITOR_LOCK_CHECK_INTERVAL));
     }
 
-    log.info("Got Monitor lock.");
+    log.info("Acquired Monitor Lock " + monitorLock.getLockPath());
```

Comment:
For what it's worth, Log4J 2.4 and higher support lambdas using a `Supplier` interface for this purpose. Java 8 java.util.logging and an SLF4J add-on also support this.

I personally find the deferred execution more clean with lambdas than with SLF4J's substitution syntax, but switching away from SLF4J in favor of something which supports suppliers would have to be a separate issue. If we didn't want to switch, we *could* create a simple wrapper facade to support deferred execution (also a separate issue, if we wanted to pursue that).